pydis_site/templates/content/articles.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'base/base.html' %}
+{% load static %}
+
+{% block title %}Articles{% endblock %}
+{% block head %}
+  <link rel="stylesheet" href="{% static "css/content/articles.css" %}">
+{% endblock %}
+
+{% block content %}
+    {% include "base/navbar.html" %}
+
+    <section class="breadcrumb-section section">
+        <div class="container">
+            <nav class="breadcrumb is-pulled-left" aria-label="breadcrumbs">
+                <ul>
+                    <li class="is-active"><a href="/articles">Articles</a></li>
```

Comment:
Use the `{% url %}` tag here:
```suggestion
                    <li class="is-active"><a href="{% url 'articles:articles' %}">Articles</a></li>
```